examples/simple.js

```diff
@@ -132,31 +140,41 @@ d3.tsv("../data/PCA_transcript_expression_TMM_RPKM_log2_sample_data.6932.tsv", f
       data.forEach(function(d) {
         d.PC1 = +d.PC1;
         d.PC2 = +d.PC2;
+        d.PC3 = +d.PC3;
```

Comment:
I'm not sure why this is hardcoded? Could we calculate this based on the number of PCA components we are going to show?